### PR TITLE
Milestone 2: Finish UI

### PR DESCRIPTION
**Pull request by mohisa302**
### Finish UI
in this milestone I:

- Finished UI
- Added some functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,151 @@
...
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Leader board <a name="about-project"></a>
+
+**Leader board**
```
Comment (franclobo) [inline README.md:30]: - **[OPTIONAL]** Kindly write about what your app is for. It helps people to use your app even if they are not programmers. To have a professional README file.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,151 @@
...
+  - Visit this link "git@github.com:mohisa302/Leader-board.git"
+  - Click the green button labelled "code"
+  - Select the "Open with GitHub Desktop" option
+  - After the GitHub Desktop add opens, click the "clone repo" button
+
+### Usage
+
+To run the project, execute the following command:
+
+**Click the "Go Live" button on your IDE(VS Code in this case)**
```
Comment (franclobo) [inline README.md]: - **[OPTIONAL]** Actually, I didn't use "Go live" please 🙏🏽 actualize your instructions, To have a professional README file. I ran:
- npm install
- npm start

File: README.md [hunk 3]
```diff
@@ -1,2 +1,151 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the whole Microverse community for their help and contributions towards this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (franclobo) [inline README.md]: - **[OPTIONAL]** If the name of your LICENSE file is LICENSE.md, please 🙏🏽 correct the link path and make sure that the link works as expected. To have a professional README file.

**Review by franclobo — APPROVED** [review]
Hi @mohisa302, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mohisa302) [reply README.md:30]: done

File: README.md (hunk 3 above)
Comment (mohisa302) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (mohisa302) [reply README.md]: done

**Pull request merged**